hopsworks-common/src/main/java/io/hops/hopsworks/common/util/Settings.java

```diff
@@ -467,6 +471,10 @@ private long setMillisecondVar(String varName, Long defaultValue) {
   private void populateCache() {
     if (!cached) {
       ADMIN_EMAIL = setVar(VARIABLE_ADMIN_EMAIL, ADMIN_EMAIL);
+      LOCALHOST = setBoolVar(VARIABLE_LOCALHOST, LOCALHOST);
+      AWS = setBoolVar(VARIABLE_AWS, AWS);
+      GCE = setBoolVar(VARIABLE_GCE, GCE);
```

Comment:
Instead of having three variables out of which two will always be false, can we have one let's say CLOUD_PROVIDER which will be AZURE, AWS or GCE?

Reply:
Ok, i did that. It also needed changes in conda-chef + hops-hadoop-chef branches.